.github/pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+# Working Group Modification Pull Request
+
+Use this template for all contributions to Projects or Governance of this Working Group.
+For membership changes (except Subproject Owner, use Modify Project below), please instead open an issue using the appropriate issue template.
+
+Use the relevant following section and delete the rest, as well as this introduction, for this Pull Request's content.
+
+# Add Project
+
+## Description
+* What is this tool?
+* Why should this tool be maintained by the Working Group?
+
+In your changes, you have added a list item to "Subprojects" in `README.md`, following this format:
+
+```
+* PROJECT_NAME
+  * Description: One sentence description of the project being added.
+  * Owners
+    * List of one or more owners in the format:
+    * Real Name @github-alias
+  * Repositories
+    * One or more repositories that are in this project as full links, e.g.
+    * https://github.com/ros-tooling/cross_compile
+```
+
+# Modify Project
+
+* What modification is being made to a Working Group Subproject?
+* Why?
+
+In your Pull Request, you have modified the relevant section.
+
+# Deprecate Project
+
+* Which Subproject are we removing from the scope of the Working Group?
+* Why?
+
+In your changes, you have removed the relevant Subproject list item.
```

Comment:
Mentioning that modification of the README in particular is what will be needed.
```suggestion
In your changes, you have removed the relevant Subproject list item from `README.md`.
```